src/Umbraco.Infrastructure/Persistence/Repositories/Implement/DataTypeRepository.cs

```diff
@@ -214,6 +214,94 @@ public IReadOnlyDictionary<Udi, IEnumerable<string>> FindListViewUsages(int id)
         return usages;
     }
 
+    public PagedModel<Guid> GetBlockEditorsReferencingContentType(Guid contentTypeId, int skip, int take)
+    {
+        // List of aliases that use elements, for now this is both block editors and the RTE.
+        var blockEditorAliases = new List<string>()
+        {
+            Constants.PropertyEditors.Aliases.BlockGrid,
+            Constants.PropertyEditors.Aliases.BlockList,
+            Constants.PropertyEditors.Aliases.RichText,
```

Comment:
While this is true from a core perspective, it does not take into account that extensions might also reference content types. Specifically custom block editors will do, but in theory - any extension can utilize a content type.